deployment/bare_metal/setup.md

```diff
@@ -95,5 +95,16 @@
 > cp .env.dist .env
 > nano .env
 >> Adjust values accordingly
+# Define name of gradido environment
```

Comment:
As I sad. A lot of overhead for little profit.

Reply:
deleted